test/objects.js

```diff
@@ -746,7 +746,6 @@
     }
     if (typeof Set === 'function') {
       var obj = new Set();
-      obj.add(1).add('string').add(false).add({});
```

Comment:
If you test code is essentially correct, but it breaks in some environments due to lack of support, then just deleting that test code is never the right solution. In this case it could have been fixed by just not chaining the calls but repeating `obj.add` four times on a separate line.